src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -13,9 +13,9 @@
 import seedu.address.commons.exceptions.IllegalValueException;
 import seedu.address.logic.commands.AddCommand;
```

Comment:
EPIC will have to maintain a list of events and persons, `AddCommand` should be refactored to `AddPersonCommand`